bin/run-addon

```diff
@@ -115,7 +115,7 @@ if [[ -z "$no_auto" ]] ; then
 fi
 
 cd build/addon
-jpm run --binary $binary --profile ../../Profile --no-copy \
+jpm run --binary $binary --profile ../../dist/Profile --no-copy \
```

Comment:
I probably never tried it from scratch, and maybe jpm doesn't work well if the profile doesn't already exist (I now vaguely recall that `--profile directory` doesn't work with jpm unless `directory` exists already).